.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+---
+
+**Description:**
+Describe your proposal.
+
+**Related issue:**
+Add issue link.
```

Comment:
I am not sure that this field makes sense. Why do we need related issue for feature request?

Reply:
Removed.